### DISCUSSION TITLE
Couldn't find 'tsserver' executable or 'tsserver.js' module

### DISCUSSION THREAD

**Discussion by mdbergmann** [joaotavora/eglot#699] [Q&A] [answered]
Install the following things using `npm` to get you running for `js-mode`:

```
npm i -g typescript typescript-language-server
```

Only installing `typescript-language-server` is not enough and will give you the above error.

**Comment by skangas** [comment, marked as answer]
Thanks! I'm replying to this just so I can mark this as "answered" in GitHub.
